apps/application-system/form/src/environments/environment.prod.ts

```diff
@@ -1,3 +1,6 @@
-export const environment = {
+export default {
   production: true,
+  identityServer: {
+    IDENTITY_SERVER_BASE_URL: 'https://identity-server.dev01.devland.is',
```

Comment:
It's still a work in progress AFAIK

Reply:
Not sure how far along they are with the prod implementation, as Atli said they are still working on it but it will most likely be `innskra.island.is`
https://github.com/island-is/helm/blob/master/charts/identity-server/values.prod.yaml#L33